runtime/shared_common/CacheMap.cpp

```diff
@@ -5191,6 +5191,16 @@ SH_CacheMap::printCacheStatsTopLayerStatsHelper(J9VMThread* currentThread, UDATA
 		CACHEMAP_PRINT1(J9NLS_DO_NOT_PRINT_MESSAGE_TAG, J9NLS_SHRC_CM_PRINTSTATS_FEATURE, "default");
 	}
 
+	if (currentThread->javaVM->jitConfig) {
+		j9tty_printf(_portlib, "\t");
+		J9SharedDataDescriptor firstDescriptor;
+		firstDescriptor.address = NULL;
+		findSharedData(currentThread, "J9AOTHeader", sizeof("J9AOTHeader") - 1, J9SHR_DATA_TYPE_AOTHEADER, FALSE, &firstDescriptor, NULL);
+		char processorFeatures[500];
+		currentThread->javaVM->jitConfig->printAOTHeader(firstDescriptor.address, processorFeatures, 500);
+		CACHEMAP_PRINT1(J9NLS_DO_NOT_PRINT_MESSAGE_TAG, J9NLS_SHRC_CM_PRINTSTATS_PROCESSOR_FEATURES, processorFeatures);
```

Comment:
Is this going to work on platforms other than X86?

Reply:
not yet on other platforms, I will guard these with J9X86 for now.

Reply:
@pshipton my `defined(J9X86)` attempt was unsuccessful, it didn't look like it was set on x86 for some reason (at least not in this file). Is there another MARCO I can use?

Reply:
J9VM_ARCH_X86 should work.

Reply:
J9X86 seems specific to Linux